lib/fluent/plugin/in_systemd.rb

```diff
@@ -55,7 +56,11 @@ def read_from
 
     def run
       watch do |entry|
-        router.emit(tag, entry.realtime_timestamp.to_i, formatted(entry))
+        begin
+          router.emit(tag, entry.realtime_timestamp.to_i, formatted(entry))
+        rescue => e
+          log.error("Exception emitting record: #{e}")
+        end
```

Comment:
Seems reasonable...

Were you seeing anything being thrown at this point that could be a bug, that we should look at?